R/utilities-project-configuration.R

```diff
@@ -19,26 +19,33 @@ createDefaultProjectConfiguration <- function(path = file.path("ProjectConfigura
 #' Creates the default project folder structure with excels file templates in
 #' the working directory.
 #'
-#' @param type A string defining the type of initialization. Currently, only `"example"`
-#' is supported and will create an example project.
+#' @param destination A string defining the path where to initialize the project.
+#' default to current working directory.
+#' @param type A string defining the type of initialization. `"new"`
+#' for folder and empty excel templates and `"example"` for an example.
 #' @export
-init_project <- function(type = "example") {
+init_project <- function(destination = ".", type = "new") {
```

Comment:
"new" is back again? I thought we agreed on not duplicating the information and not providing an "empty" project so far.

Reply:
Indeed, mistake during merge conflict resolution.

fixed.